### ISSUE TITLE
Yauheniya: testing, week 2

### ISSUE THREAD

**Issue by yauheniya-askolkava** [HackYourFutureBelgium/class-9-10#506]
# Wednesday Check-In

__testing, week 2__

## Progress

What have you learned and/or completed so far this week?
It is getting to be systematic, which I really like. It helps me understand the code, not fully, but I grasp quite a lot; found document and pass a good one in order to understand the logic.

## Blocked

What are you having trouble understanding? What parts of the homework are you struggling to complete?
Nothing really, just timing, I feel I am letting down my groupwork, but I can not be effective there if I do not study on my own. It takes time. Hope they are not grumpy )

## Next Steps

What are your goals and your study plans for the rest of the week?
Will continue investigate the codewars + group_work.

## Tip(s) of the week

Share some tips, tricks or links that made a big difference for you.

**Comment by colevandersWands** [comment]
> I feel I am letting down my groupwork, but I can not be effective there if I do not study on my own. It takes time. 

You're right on about needing to practice before contributing to the harder parts of a project, but there is much more going in a project than just the JavaScript.  you can let you team know what you're comfortable with and just contribute how you can.  Maybe you do the html/css, or are the repository owner.

**Comment by yauheniya-askolkava** [comment]
Thank you, Evan! I indeed try to be as much transparent and honest I can!
